Plugins/NativeEngine/Source/NativeEngine.cpp

```diff
@@ -1021,6 +1022,13 @@ namespace Babylon
             });
     }
 
+    void NativeEngine::CopyTexture(const Napi::CallbackInfo& info)
+    {
+        const auto textureDestination = info[0].As<Napi::External<TextureData>>().Data();
```

Comment:
is there any clean up required for the original texture this data's handle was referencing?

Reply:
that's part of the 'copy texture' task on this list : https://github.com/BabylonJS/BabylonNative/issues/616#issuecomment-831162396

Reply:
It would be good to put a comment here so that people know this code is not correct.